### PR TITLE
Support the Casbin authorization method.

### PR DESCRIPTION
**Pull request by hsluoyz**
This PR solves: https://github.com/cesanta/docker_auth/issues/178

---
This change is [Reviewable](https://reviewable.io/reviews/cesanta/docker_auth/182)

### CODE REVIEW

**Comment by hsluoyz** [comment]
Hi @rojer , any update on this PR?

**Review by techknowlogick — COMMENTED** [review]
Thanks for this PR @hsluoyz. Could you rebase this onto latest commit in master, add copyright headers to top of files, and also add some documentation?

File: auth_server/authz/casbin_authz_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The casbin Authors. All Rights Reserved.
```
Comment (hsluoyz) [inline auth_server/authz/casbin_authz_test.go:1]: 2017 -> 2021

**Comment by hsluoyz** [comment]
Hi @techknowlogick , we have fixed what your proposed. Please review.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
This is wonderful, thank you so much!!

**Pull request merged**